### ISSUE TITLE
Automatic second selection, even if seconds are disabled

### ISSUE THREAD

**Issue by maltemedocs** [subhamayd2/day_night_time_picker#111]
When an hour was selected, the picker immediately continues to the minute selection. This works. But then, when the minutes were selected, the picker immediately deselects the minutes and slider goes to the left. I assume it goes on to the seconds selection, which can't work when the seconds are not active and makes the plugin feel broken. This happens on the current version, 1.2.0+2.

Since this felt hard to describe, I'll attach a video showing the buggy behaviour here:

https://github.com/subhamayd2/day_night_time_picker/assets/71251334/ba8b14c7-96e3-4de0-b421-011c7b31d13e

It's not from a minimal example, but I don't think we do anything strange with the plugin.

**Comment by nwgiles** [comment]
Got the same issue here. We don't do anything special either. 
This picker is great but can't use it until this is resolved.

```
showPicker(
  context: context,
  value: getTime(),
  onChange: onTimeChanged,
  minuteInterval: TimePickerInterval.FIVE,
  showSecondSelector: false
)
```

**Comment by maltemedocs** [comment]
@nwgiles The version 1.1.6 is the last version without the seconds change and still works with the newest flutter version, might be a workaround also for you, until this is fixed.

**Comment by subhamayd2** [comment]
Hi guys, thanks for using the package. This bug is fixed in the latest release (v1.3.0).
